sdk/communication/communication-short-codes/src/utils/url.ts

```diff
@@ -0,0 +1,4 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+export { URL, URLSearchParams } from "url";
```

Comment:
url.ts and url.browser.ts are not required. the types are supported by node:

https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams
https://developer.mozilla.org/en-US/docs/Web/API/URL

Reply:
The files `url.ts` and `url.browser.ts` are needed to avoid having platform checks to decide where to import from. When running in node you need to import from `"url"` whereas for browsers it is in the global scope. This allows you to always import from `"utils/url.ts"` and rely on the browser mapping to get the right one

Reply:
They're global in Node now, in all nodes we support. So it's not needed anymore